server/apiv2/handlers/tso_keyspace_group.go

```diff
@@ -35,6 +35,8 @@ func RegisterTSOKeyspaceGroup(r *gin.RouterGroup) {
 	router.GET("", GetKeyspaceGroups)
 	router.GET("/:id", GetKeyspaceGroupByID)
 	router.DELETE("/:id", DeleteKeyspaceGroupByID)
+	router.POST("/split", SplitKeyspaceGroupByID)
```

Comment:
No need `:id`? And how about `/:id/split`

Reply:
ref: https://docs.github.com/en/rest/gists/gists?apiVersion=2022-11-28#fork-a-gist